.travis.yml

```diff
@@ -38,30 +33,32 @@ install:
   - nvm use $TRAVIS_NODE_VERSION
 
   # Install requirements
-  - pip install -r requirements_devel.txt
+  - pip install -r requirements.txt
   - npm install
+  - pip install coveralls
   - if [[ "$DB" = "postgresql" ]]; then pip install psycopg2; fi
```

Comment:
We seem to be installing psycopg2 in the requirements.txt. Is it necessary to have this line of code? Also, since the database being used is certainly Postgres, wouldn't be better to remove the conditional installation of psycopg2?

Reply:
You can remove kit since they are doing the same job

Reply:
Noted
👍🏽

Reply:
Good point. 👍🏽